### PR TITLE
make support section

### PR DESCRIPTION
**Pull request by nibchao**
unknown

the three buttons have temporary urls

i deleted my schedule section files from my vs code directory and pushed this file deletion to my support section branch. 

i'm not sure if merging this support section pull request will delete the schedule section files in the main dev repo/branch when my first pull request for the schedule section gets merged.

### CODE REVIEW

File: components/Support/Support.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          like to help hackers throughout the day.
+        </h2>
+        <p className="text-left">
+          <p className="font-medium">Mentors</p>
+          <p>
+            Mentors are in charge of helping the hackers with new technologies
+            and working through any bugs or obstacles they encounter. We
+            recommend this position if you are well versed in a particular stack
+            to offer the best guidance.
+          </p>
```
Comment (minsooerickim) [inline components/Support/Support.tsx:21]: can u make each of these little sections ("mentors", "volunteers", etc.) a component, and pass in the details in as a parameter

**Comment by minsooerickim** [comment]
remove yarn.lock file

**Comment by minsooerickim** [comment]
and yea idk what u did with git lol it looks like ur commits r a bit messed up but ur files changed looks fine so it shud be fine

**Comment by nibchao** [comment]
> and yea idk what u did with git lol it looks like ur commits r a bit messed up but ur files changed looks fine so it shud be fine

ah i see what i did wrong, i made a new branch from my schedule-section branch instead of making it from my dev branch

**Comment by nibchao** [comment]
should i close this pull request, make a new branch from my dev branch, add only the support-section files + make your suggested changes, then make a new pull request? or keep this pull request even though you said my commits are kinda messed up? @minsooerickim

**Comment by nibchao** [comment]
i'm making a new pull request because i made a new branch from my schedule-section branch instead of making a new branch from my dev branch which messed up the commit history

**Pull request closed without merging**